docs/sources/explore/index.md

```diff
@@ -155,6 +155,31 @@ Along with metrics, Explore allows you to investigate your logs with the followi
 
 Results of log queries are shown as histograms in the graph and individual logs are displayed below. If the data source does not send histogram data for the requested time range, the logs model computes a time series based on the log row counts bucketed by an automatically calculated time interval and the start of the histogram is then anchored by the first log row's timestamp from the result. The end of the time series is anchored to the time picker's **To** range.
 
+#### Log level
+If log has a **level label**, we use the value of the label to determine log level and update color accordingly. If log doesn't have a level label, we parse the log line to determine log level. If Grafana is not able to determiine log level, it will be visualised with **unknown log level**.
```

Comment:
- should be `**level** label`, because "level" is the important label.
- should be `**unknown** log level`. 
- 1st sentence: "If the logs have a..."
- 2nd sentence: "If the logs do not have..."
- 3rd: "If Grafana is not able to determine the log ..."

Reply:
Great points! Updated. 🙂